rules/linux/auditd/lnx_auditd_bpfdoor_port_redirect.yml

```diff
@@ -0,0 +1,28 @@
+title: BPFDoor_TCP_ports_redirect
```

Comment:
Can you please remove the `_` from the title and make every first letter a capital letter?